### PR TITLE
Add initial stubs and cause model for cancer causes

### PR DESCRIPTION
**Pull request by kiranklc**

### CODE REVIEW

File: docs/source/gbd2017_models/causes/neoplasms/bladder_cancer/cancer_model.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+.. todo::
+
+   Add assumptions and limitations.
+
+
+Cause Model Diagram
+-------------------
+
+According to GBD 2017, cancer cases are considered cured if the patient survives more than 10 years. But, there is no data available for remission rate
```
Comment (beatrixkh) [inline docs/source/gbd2017_models/causes/neoplasms/bladder_cancer/cancer_model.rst]: I would consider clarifying this language. does this mean that GBD uses a duration-based remission rate? I would also clarify between "according to gbd, the world says.." vs "within GBD, this is defined as..."

File: docs/source/gbd2017_models/causes/neoplasms/cancers/cancer_model.rst [hunk 2]
```diff
@@ -0,0 +1,131 @@
+.. _2017_cancer_model_cancers:
+
+=======
+Cancers
+=======
+
+This is a common document for the cancers that are modelled similarly in GBD. This document currently has information related to the following cancers:
```
Comment (beatrixkh) [inline docs/source/gbd2017_models/causes/neoplasms/cancers/cancer_model.rst]: does this mean that they're also going to have the same model structure within vivarium?

File: docs/source/gbd2017_models/causes/neoplasms/index.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.. _2017_cause_neoplasms:
+
+==============================
+Neoplasm (Cancer) Cause Models
```
Comment (beatrixkh) [inline docs/source/gbd2017_models/causes/neoplasms/index.rst]: is this page eventually going to have some of the additional headings/structure of the other pages? (experts v gbd v vivarium?)

Comment (beatrixkh) [reply docs/source/gbd2017_models/causes/neoplasms/index.rst]: ohh i was misinterpreting; pls disregard

File: docs/source/gbd2017_models/causes/neoplasms/cancers/cancer_model.rst (hunk 2 above)
Comment (kiranklc) [reply docs/source/gbd2017_models/causes/neoplasms/cancers/cancer_model.rst]: yes

File: docs/source/gbd2017_models/causes/neoplasms/bladder_cancer/cancer_model.rst (hunk 1 above)
Comment (kiranklc) [reply docs/source/gbd2017_models/causes/neoplasms/bladder_cancer/cancer_model.rst]: No, they don't. They have used a complicated model. I will change the wording.

**Comment by NathanielBlairStahn** [comment]
@kiranklc Have you built the html with Sphinx to make sure everything shows up in the table of contents?  The existing TOC pages are written to detect files called index.rst, but you have named your files cancer_model.rst

**Comment by NathanielBlairStahn** [comment]
@kiranklc nvm, I just built it myself and it looks like it works.  I think everything looks good.

**Comment by kiranklc** [comment]
It works like a hierarchy. Under neoplasms page, different cancers will appear. The main page is named index and sub pages cancer_model.rst

**Pull request merged**